gems/sorbet/test/snapshot/partial/db_schema/expected/out.log

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
Given my comment below about foo.txt, i'd recommend uncommitting out.log in expected.

Reply:
I've removed it.